index.html

```diff
@@ -60,5 +60,172 @@ <h2>I’m Ahmed Glad to see you!</h2>
+            <div class="js"><p class="second-project-js">javaScript</p></div>
+          </div>
+          <div class="action">
+            <button><p class="second-button">See Project</p></button>
+          </div>
+        </div>
+      </div>
+      <div class="card-3">
+        <img src="assets/images/Snapshoot Portfolio-3.png" alt="Snapshoot-portfolio">
+        <div class="info-block">
+          <div class="client">
+            <h2 class="project-title-3">Tonic</h2>
+            <div class="details-3">
+              <div class="name"><p class="third-project-name">CANOPY</p></div>
+              <div class="circle-1"></div>
+              <div class="role"><p class="third-project-role">Back End Dev</p></div>
+              <div class="curcle-2"></div>
+              <div class="year"><p class="third-project-year">2015</p></div>
+            </div>
+          </div>
+          <p class="summary-3">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <div class="languages-3">
+            <div class="html"><p class="third-project-html">html</p></div>
+            <div class="css"><p class="third-project-css">css</p></div>
+            <div class="js"><p class="third-project-js">javaScript</p></div>
+          </div>
+          <div class="action">
+            <button><p class="third-button">See Project</p></button>
+          </div>
+        </div>
+      </div>
+      <div class="card-4">
+        <img src="assets/images/Snapshoot Portfolio-4.png" alt="Snapshoot-portfolio">
+        <div class="info-block">
+          <div class="client">
+            <h2 class="project-title-4">Multi-Post
+              Stories</h2>
+              <div class="details-4">
+                <div class="name"><p class="forth-project-name">CANOPY</p></div>
+                <div class="circle-1"></div>
+                <div class="role"><p class="forth-project-role">Back End Dev</p></div>
+                <div class="curcle-2"></div>
+                <div class="year" class="forth-project-year"><p>2015</p></div>
+              </div>
+          </div>
+          <p class="summary-4">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <div class="languages-4">
+            <div class="html"><p class="forth-project-html">html</p></div>
+            <div class="css"><p class="forth-project-css">css</p></div>
+            <div class="js"><p class="forth-project-js">javaScript</p></div>
+          </div>
+          <div class="action">
+            <button><p class="forth-button">See Project</p></button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="About-my-self">
+      <div class="top">
+        <h2 class="About">About
+          Myself</h2>
+        <p class="about-my-self-paragraph">Hello I’m a software developer!
+          I can help you build a product ,
```

Comment:
- [ ] :thinking: Kindly make sure your About section design matches the Figma's, using the same fonts and adding the top-right radius:recycle: while manipulating the margins and paddings. 
- [ ] :thinking: Kindly make sure the stacks section also looks like the design by using the same fonts and centering the content, making sure the arrows up and down do not have a different background than the whole section :arrow_down: :arrow_right: 

-Your design:

![Screenshot from 2023-01-19 18-30-11](https://user-images.githubusercontent.com/97834160/213503911-2e907fbb-9f3d-4395-af57-fc6526da6bda.png)


![Screenshot from 2023-01-19 18-44-48](https://user-images.githubusercontent.com/97834160/213504527-8f6567f0-de27-4852-adfb-efc0de67ba3b.png)



-Figma:

![Screenshot from 2023-01-19 18-30-33](https://user-images.githubusercontent.com/97834160/213504410-651507d6-88f4-4f62-b390-d64285ee99ac.png)

![Screenshot from 2023-01-19 18-44-48](https://user-images.githubusercontent.com/97834160/213504565-9c5e5a39-d006-42ba-a24c-94a454d604bd.png)